README.md

```diff
@@ -329,6 +329,10 @@ strings that they parse.
   (`major`, `premajor`, `minor`, `preminor`, `patch`, `prepatch`, or `prerelease`),
   or null if the versions are the same.
 
+### Comparators
+* `intersects(comparator)`: Return true if the comparators intersect
+* `satisfiesRange(range)`: Return true if the comparator intersects with any
```

Comment:
The naming of this is a bit weird.  You wouldn't normally say that a comparator "satisfies" a range.  A range is satisfied by a single _thing_ which has a specific version, not by a comparator, which is like a mini-range.  Also, a comparator like `<=1.4.5` would return true for this method, given the Range `>=1.3.0`, even though many versions allowed by that comparator are not satisfying answers for the Range.  We can only talk about intersections and unions with ranges (and their building blocks, Comparators) because they are sets, not discrete atomic items.

Having a `comparator.intersects(comparator)` method seems like a building block towards having range intersection.  People generally don't interact with comparators directly.  Having (effectively) `comparator.intersects(range)` is a bit odd, and using the language of "satisfying" makes it odder.

It seems like what you'd really want in most cases is `range.intersects(range)`.  It's probably better to leave aside things that don't get us closer to that.